region/lowernorfair/west/Screw Attack Room.json

```diff
@@ -611,6 +611,131 @@
       ],
       "endsWithShineCharge": true
     },
+    {
+      "link": [1, 4],
+      "name": "Side Platform Jump with Blue Speed",
+      "entranceCondition": {
+        "comeInWithSidePlatform": {
+          "platforms": [
+            {
+              "minHeight": 1,
+              "maxHeight": 1,
+              "minTiles": 17,
+              "speedBooster": true,
+              "obstructions": [[1, 0]],
+              "requires": [
+                {"getBlueSpeed": {
+                  "usedTiles": 15,
+                  "openEnd": 1
+                }},
+                {"or": [
+                  "canMomentumConservingMorph",
```

Comment:
It feels to me like it is an airball and not really ceilingMorphTech.  I'm morphing after the door.